configs/milvus.yaml

```diff
@@ -979,6 +979,14 @@ quotaAndLimits:
       enabled: false # switch to enable l0 segment row count quota
       lowWaterLevel: 32768 # l0 segment row count quota, low water level
       highWaterLevel: 65536 # l0 segment row count quota, low water level
+    deleteBufferRowCountProtection:
+      enabled: false # switch to enable delete buffer row count quota
+      lowWaterLevel: 32768 # delete buffer row count quota, low water level
+      highWaterLevel: 65536 # delete buffer row count quota, high water level
+    deleteBufferSizeProtection:
+      enabled: false # switch to enable delete buffer size quota
+      lowWaterLevel: 32768 # delete buffer size quota, low water level
+      highWaterLevel: 65536 # delete buffer size quota, high water level
```

Comment:
The default values may be too small, I suppose it won't be a problem if the deletions can be easily consumed in less than 1 minute or so.

And, please note the unit of the delete buffer size in the comments.